client/components/view-full-recipe.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import checkDate from './checkdate';
+
+function splitLines(array) {
+  return (
+    // eslint-disable-next-line array-callback-return
+    array.map((item, i) => {
+      if (item) {
+        return <li key={i}>{item}</li>;
+      }
+    })
+  );
+}
+
+const back = () => {
+  history.back();
+};
+
+function TaglistFromArray(array) {
+  return (
+    // eslint-disable-next-line array-callback-return
+    array.map((item, i) => {
+      if (item) {
+        return <a className='tags' key={i}>{item}</a>;
+      }
```

Comment:
After the if you can add a `return null` to fix the linter issue

Reply:
Got it, thanks